src/ontology/imports/pato_import.owl

```diff
@@ -5451,14 +6014,20 @@ For example, A and B may be gene products and binding of B by A positively regul
         <owl:annotatedTarget>A cell morphology that inheres in neurons which are small and round, and make highly specialized synapses with the axon initial segments of pyramidal neurons, wrapping them with candle-shaped synaptic “cartridges”.</owl:annotatedTarget>
         <oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string">doi:10.1016/b978-0-12-369497-3.10004-4</oboInOwl:hasDbXref>
     </owl:Axiom>
+    <owl:Axiom>
+        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/PATO_0070011"/>
+        <owl:annotatedProperty rdf:resource="http://www.w3.org/2000/01/rdf-schema#comment"/>
+        <owl:annotatedTarget>Chandelier cells can be bitufted or multipolar.</owl:annotatedTarget>
+        <oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string">DOI:10.1016/B978-0-12-369497-3.10004-4</oboInOwl:hasDbXref>
+    </owl:Axiom>
     
 
 
     <!-- http://purl.obolibrary.org/obo/PATO_0070012 -->
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/PATO_0070012">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/PATO_0010006"/>
-        <obo:IAO_0000115>A cell morphology that inheres in neurons which have tufts of dendrites extending from two opposite poles of the soma, and have axons that form a plexus which spreads widely, usually descending.</obo:IAO_0000115>
+        <obo:IAO_0000115>A cell morphology that inheres in neurons which have two clusters of branches that originate directly from the soma and extend in opposite directions and axons that form a plexus which spreads widely. Compared to bipolar morphology, bitufted morphology has branching that occur close to the soma.</obo:IAO_0000115>
```

Comment:
1. Are branches only dendritic branches?
2. It should be 'occurs' instead of 'occur' in the gloss.

Reply:
@patrick-lloyd-ray not quite sure what you mean by are branches only dendritic branches? I think I was aiming to talk about two sets of branches separately - the dendrites which are free branching and contains smooth spines and the axons which are highly branched and give rise to basket-like structures, not sure if that came out well in the def.

Reply:
sorry for the previous reply (which I've deleted) was looking at the wrong thing. 

This current def is pulled from PATO, so fixes will have to be done there. 
Will create a ticket there and link it :) 

Thanks